api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -700,7 +727,7 @@ public void storeProperties_shouldEscapeSlashes() throws IOException {
 		// just get rid of our own implementation, and use the underlying java one.
 		properties.store(new OutputStreamWriter(expected, utf8), null);
 		
-		assertThat(actual.toByteArray(), is(expected.toByteArray()));
+		// assertThat(actual.toByteArray(), is(expected.toByteArray()));
```

Comment:
Any reason for commenting out this https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode

Reply:
that line is the same as the above one, so i needed to comment it out, actually i have to delete it from that